ios/FluentUI/Vnext/List/ListCell.swift

```diff
@@ -469,7 +480,7 @@ struct ListCellButtonStyle: ButtonStyle {
             return Color(stateBackgroundColor)
         }()
 
-        if isPressed {
+        if isPressed || state.isSelected {
```

Comment:
there should be a gigantic todo here right? normally, i would consider background color staying grey on the cell as a bug rather than proper selected state.

Reply:
I mentioned in the PR description that I'd follow up with the correct styling, but I can add a todo. I didn't add it this time because I was going to have a PR out for that within a day or two of this closing.